hardhat.config.ts

```diff
@@ -122,6 +146,8 @@ const config: HardhatUserConfig = {
     mainnet: getChainConfig("mainnet"),
     testnet_bsc: getBSCConfig("testnet"),
     mainnet_bsc: getBSCConfig("mainnet"),
+    mumbai: getPolygonConfig("testnet"),
```

Comment:
maybe call it testnet_polygon?

Reply:
according to the function name + parameter, we can see that it is a "testnet polygon config" :D

Reply:
Since there is only one testnet chain for polygon, that has also a known name, i would keep it like that.